### DISCUSSION TITLE
collaborator们可以不可以自己交的issue自己换标签……

### DISCUSSION THREAD

**Discussion by Deep-Dark-Forest** [Meloong-Git/PCL#3320] [提问] [answered]
问题一：可不可以自己给自己开的issue更换标签？（比如直接把新反馈改成待处理）
问题二：可不可以在开issue之前就添加个删除标签？

**Comment by iPodtouch6557** [comment]
你怎么会想这种问题~法外狂徒张三白糖当砒霜卡法律的bug是吧~
理论应该可行，但是没有人做就是了，猜的

**Reply by Deep-Dark-Forest** [reply]
额抱歉我表达有点问题，标题和正文都要回答（

**Comment by WForst-Breeze** [comment]
可以

**Reply by Deep-Dark-Forest** [reply]
额抱歉我表达有点问题（
麻烦您重新看一下

**Reply by WForst-Breeze** [reply, marked as answer]
都可以
![image](https://github.com/Hex-Dragon/PCL2/assets/110760354/086e49c2-129b-413c-8175-d4098b615a66)

**Reply by Deep-Dark-Forest** [reply]
唔，我想说的不是“有没有这么做的权限”而是“pcl2社区是否允许这么做”，因为我未曾看到过有collaborator这么做过……
非常抱歉我表达有问题

**Reply by WForst-Breeze** [reply]
那就是第一个可以，第二个可以但没必要（）一般删除标签是对有破坏性内容和涉及安全性内容才会加的，不是想加就能加的

但一般来说 Collaborator 不会直接给自己加待处理，一般都会等有人再次确认过一遍再去加，这样保险一些

**Comment by allMagicNB** [comment]
1.可以
2.可以但有病

**Reply by hejiehao** [reply]
如果在开issue之前就添加个删除标签会不会把权限撤掉

**Reply by allMagicNB** [reply]
> 如果在开issue之前就添加个删除标签会不会把权限撤掉

哪个权限？

**Reply by Deep-Dark-Forest** [reply]
> > 如果在开issue之前就添加个删除标签会不会把权限撤掉
> 
> 哪个权限？

改标签的权限（可能吧）

**Reply by hejiehao** [reply]
collaborator 的权限

**Reply by allMagicNB** [reply]
> collaborator 的权限

你这得看龙猫怎么想（

**Reply by Light-Beacon** [reply]
> 如果在开issue之前就添加个删除标签会不会把权限撤掉

我不信现在的哪个 Collaborator  会这么乱玩

**Reply by hejiehao** [reply]
~~我建议以身试法~~
